spec/models/user_routine_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe UserRoutine, type: :model do
+  before :each do
+    user = User.create(name: 'Jim')
+    routine = Routine.create(name: 'Leg Day')
+    @user_routine = UserRoutine.create(routine: routine, user: user, date: Date.new(2019,5,20))
```

Comment:
What again is the difference between an object created with an @ vs without?

Reply:
The @ creates an instance variable that allows it to be called within the larger scope. Without it, the variables are scoped to the block. I'm only using the user and routine variable in line 7, so I didn't need an @ .